ocaml/sdk-gen/common/CommonFunctions.ml

```diff
@@ -197,6 +197,16 @@ let gen_param_groups message params =
        (if expRelease = "" then overloadGroups else [params])
     )
 
+and get_minimum_allowed_role msg =
```

Comment:
This is probably a question for the Toosltack folk: Should we exclude internal roles?

I suppose this also goes for the API docs, which currently do not hide internal roles. For example, the mimimun role for `repository.set_name_label` is documented as `client-cert` - not sure how much sense this makes to the API users. (Note that on the UI we do hide internal roles).

Reply:
cc @psafont @lindig

Reply:
I think it makes sense as a policy to hide internal roles as a policy. Might as well start now here